src/Metadata/class-metadata-builder.php

```diff
@@ -120,6 +120,21 @@ protected function build_type( WP_Post $post, string $parsely_type ): void {
 		$this->metadata['@type'] = $type;
 	}
 
+	/**
+	 * Populates all the fields related to post ID.
+	 *
+	 * @param WP_Post $post The post/page for which to populate the field.
+	 *
+	 * @since 3.0.x
+	 */
```

Comment:
```suggestion
	/**
	 * Populates the custom metadata field with the current post's/page's ID.
	 *
	 * @since 3.16.0
	 *
	 * @param WP_Post $post The post/page for which to populate the field.
	 */
```